core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -0,0 +1,344 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import java.net.URL
+import java.time.Instant
+import java.util.UUID
+
+import cats.implicits._
+import enumeratum.{Enum, EnumEntry}
+import org.broadinstitute.dsde.workbench.google2.MachineTypeName
+import org.broadinstitute.dsde.workbench.leonardo.DataprocRole.SecondaryWorker
+import org.broadinstitute.dsde.workbench.leonardo.RuntimeContainerServiceType.JupyterService
+import org.broadinstitute.dsde.workbench.leonardo.RuntimeImageType.{CustomDataProc, Jupyter, RStudio, Welder}
+import org.broadinstitute.dsde.workbench.model.google.{parseGcsPath, GcsBucketName, GcsPath, GoogleProject}
+import org.broadinstitute.dsde.workbench.model.{ValueObject, WorkbenchEmail}
+
+import scala.collection.immutable
+
+/**
+ * This file contains models for Leonardo runtimes.
+ */
+/** The runtime itself */
+final case class Runtime(id: Long = 0, // DB AutoInc
```

Comment:
can we remove the default value for `id`?

Reply:
Would be nice.. I can try.